__tests__/app.test.js

```diff
@@ -105,4 +106,42 @@ describe('GET /api/articles', () => {
                 })
             })
     })
+})
+describe('GET /api/articles/:article_id/comments', () => {
+    it('return status 200 and an array of comments for the given article_id', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            const expectedKeys = ["comment_id", "votes", "created_at", "author", "body", "article_id"]
+            expect(body.comments).toBeSortedBy("created_at", { descending: true })
+            body.comments.forEach((comment) => {
+                expect(Object.keys(comment)).toEqual(expect.arrayContaining(expectedKeys))
+            })
+        })
+    })
+    it('return status 200 even when there are no comments for an existing article', () => {
+        return request(app)
+        .get('/api/articles/2/comments')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.msg).toBe('No comments found for this article');
```

Comment:
If the article has no comments, I wouldn't expect an error message like this, rather an empty array